src/sql/workbench/services/dialog/browser/customDialogService.ts

```diff
@@ -20,6 +20,12 @@ export class CustomDialogService {
 
 	public showDialog(dialog: Dialog, dialogName?: string, options?: IModalOptions): void {
 		let name = dialogName ? dialogName : 'CustomDialog';
+
+		if (options && (options.dialogStyle === 'callout' || options.dialogStyle === 'calloutCompact')) {
+			options.positionX = document.activeElement.getBoundingClientRect().left;
+			options.positionY = document.activeElement.getBoundingClientRect().top;
+			options.renderFooter = false;
```

Comment:
if callout will always default to renderFooter = false, this logic should be added to modal.ts instead of here, right?